src/unix/bsd/freebsdlike/dragonfly/mod.rs

```diff
@@ -1589,6 +1589,11 @@ f! {
     pub fn minor(dev: ::dev_t) -> ::c_int {
         (dev & 0xffff00ff) as ::c_int
     }
+
+    #[deprecated(since = "0.2.77", note = "Use `__errno_location()` instead")]
+    pub fn __error() -> *mut ::c_int {
+        super::__errno_location()
+    }
```

Comment:
We could simply remove this function on the 1.0 release (i.e. main branch).

Reply:
Done